modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/tabs/ActionBuilder.tsx

```diff
@@ -230,6 +236,19 @@ export default function ActionBuilder({
 				</ClayAlert>
 			)}
 
+			{hasLocalizedField && (
```

Comment:
Should these changes be under feature flag? I'm not aware since I'm not as much in the context of this as you  guys are @cgmonte @marcelabc. can you confirm here?
If so, here you can include the FF validation here as well, like this:
`{Liferay.FeatureFlags['LPD-32050'] && hasLocalizedField && (`

If not, please disregard this!

Reply:
I discussed it with Carlos and we agreed that the changes should be flagged, thank you!